docs/source/insights_upload_configuration.rst

```diff
@@ -0,0 +1,63 @@
+Installing the Insights Client
+------------------------------
+To work with the Insights Client, we must also install the Insights Core. To begin, create the insights directory at the same level as quipucords and clone the following repositories::
+
+    mkdir insights && cd insights
+    git clone git@github.com:RedHatInsights/insights-core.git
+    git clone git@github.com:RedHatInsights/insights-client.git
+
+Setting Up a Virtual Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The insights-client will need to be installed inside of the same virtual environment as the QPC client::
+
+    cd ../location/of/qpc/client
+    pipenv shell
+
+Edit the Insights Client Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You will need modify the Insights Client Configuration in order to be authorized to upload::
+
```

Comment:
Note that you need to uncomment values.  Seems obviously but the docs are for onboarding new team members so extra details are helpful.